bemani/backend/iidx/pendual.py

```diff
@@ -1185,6 +1190,35 @@ def handle_IIDX22pc_request(self, request: Node) -> Optional[Node]:
             elif timeshift_override == 3:
                 common_timeshift_phase.set_attribute('phase', '2')
 
+            root.add_child(Node.void('eaappli_expert'))
+
+            cc_collabo_event = Node.void('cc_collabo_event')
+            root.add_child(cc_collabo_event)
+            if ccxbm_on:
+                cc_collabo_event.set_attribute('phase', "1")
+                root.add_child(Node.void('cc_collabo_license'))
+            else:
+                cc_collabo_event.set_attribute('phase', "0")
+
+            tohoisfumo = Node.void('toho_remix')
+            root.add_child(tohoisfumo)
+            tohoisfumo.set_attribute('phase', "2")
+
+            cm = Node.void("cm")
+            root.add_child(cm)
+            cm.set_attribute('compo', 'cm_event3')
+            cm.set_attribute('folder', 'cm_event3')
+            cm.set_attribute('id', "0")
+
+            #Ranking details in title screen
```

Comment:
Nice.

Reply:
these are separate: comment goes to the rankings under, though this is not actually what they're for or im missing part of the puzzle. was meant to be looked at again, had not gotten to that.